src/qibocal/protocols/characterization/allxy/allxy.py

```diff
@@ -88,11 +93,17 @@ def _acquisition(
             )
 
         # execute the pulse sequence
-        results = platform.execute_pulse_sequence(sequence)
+        results = platform.execute_pulse_sequence(
+            sequence,
+            ExecutionParameters(
+                nshots=params.nshots,
+                averaging_mode=AveragingMode.CYCLIC,
```

Comment:
```suggestion
                averaging_mode=AveragingMode.SINGLESHOT,
```
This should be singleshot right? Otherwise probabilities do not really make sense (also for other allxy routines)

Reply:
You will get a value between 0 and 1 from the averaging. I don't see a problem with that.

Reply:
Here we are averaging and classifying, maybe the problem is that I first average and than classify while you first classify and then average...
But then in the StateResult there will be something that is not 0-1 (if you average after classification) and I thought that this is not wanted

Reply:
Yes, I do first classification and the average. And also, the AveragedStateResults are expected not to be 0 or 1. Just a value in between so the results are the same as the old AllXY